### PR TITLE
Fixed search bar on homepage and template page and fixed hover effect on navigation tab

### PR DESCRIPTION
**Pull request by Abdel29111**

### CODE REVIEW

File: src/components/home/search/constants.ts [hunk 1]
```diff
@@ -5,7 +5,15 @@ import Template4 from '@/assets/image/template-4.png';
...
+    imgSrc: Template2,
+    label: 'Application security template',
+    link: 'www.instagram.com',
+  },
+  {
+    imgSrc: Template3,
+    label: 'App registration template',
+    link: 'www.instagram.com',
+  },
+  { imgSrc: Template4, label: 'Himzo', link: 'www.instagram.com' },
```
Comment (kovacevicselma) [inline src/components/home/search/constants.ts:18]: who is Himzo?

File: src/components/home/search/search.tsx [hunk 2]
```diff
@@ -25,24 +21,22 @@ import { useStyles } from './styles';
...
-  );
+  const filteredTemplates = TEMPLATES.filter((template) => {
+    return template.label.toLowerCase().includes(searchField.toLowerCase());
+  });
 
   function handleOpenTemplates(): void {
     navigate('/homepage/templates');
   }
 
+  function handleChange(e: { target: { value: SetStateAction<string> } }) : void {
```
Comment (kovacevicselma) [inline src/components/home/search/search.tsx:36]: You can simply change to
function handleChange({ target: { value } }: React.ChangeEvent<HTMLInputElement>): void {
  setSearchField(value);
};

File: src/components/templates/nav-item/styles.ts [hunk 3]
```diff
@@ -7,6 +7,22 @@ export const useStyles = tss.create(({ theme }) => {
       alignItems: 'center',
       gap: '10px',
       cursor: 'pointer',
+      width: '256px',
+      height: '40px',
+      paddingLeft: '15px',
+      paddingRight: '10px',
+      '&:hover': {
```
Comment (kovacevicselma) [inline src/components/templates/nav-item/styles.ts:14]: Move hover to the end

Comment (Abdel29111) [reply src/components/templates/nav-item/styles.ts:14]: Hover is already at the bottom of navItem style.

**Pull request merged**